shortstories/templates/shortstories/shortstories_edit.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base/index.html' %}
+
+{% block content %}
+    <h1>New Short Story</h1>
+    <form method="POST" class="post-form">{% csrf_token %}
```

Comment:
Does this form work? i.e. do you see the result of submit in your database or shortstories index view?

Reply:
I do testing now